packages/element-lib/src/sidetree-v2/op.js

```diff
@@ -5,11 +5,13 @@ const getDidDocumentModel = (primaryPublicKey, recoveryPublicKey) => ({
   publicKey: [
     {
       id: '#primary',
+      usage: 'signing',
       type: 'Secp256k1VerificationKey2018',
       publicKeyHex: primaryPublicKey,
     },
     {
       id: '#recovery',
+      usage: 'recovery',
```

Comment:
note: these properties need to be defined in our own JSON-LD context or they will cause interop issues.

Reply:
I have created: https://github.com/decentralized-identity/element/issues/115